committee-steering/governance/working-group-formation-process.md

```diff
@@ -0,0 +1,87 @@
+# Kubernetes Working Group Formation Process - Proposal
+
+Draft 1.0 // Jaice Singer DuMars // June, 2018
+
+## Process Overview and Motivations
+As opposed to “Birds of a Feather” or discussion groups, Working Groups provide a more formal avenue for disparate
+groups to intellectually collaborate around a common problem, craft a balanced position, and disband. Because they
+represent the interests of multiple special interest groups, they are a vehicle for consensus building, not
+construction. By design, the code under discussion resides in either multiple parts of the project, or is in the
+process of being extracted to an external repository.
+
+Because a Working Group is an acknowledged part of the Kubernetes governance model, a process is necessary for
+formation and disbanding, as well as guidance on what authority is vested in the group by the steering committee.
+
+## Goals of the process
+
+- An easy-to-navigate process for those wishing to establish and eventually disband a new Working Group
+- Simple guidance and differentiation on where a Working Group makes sense, and does not
+- Clear understanding that no authority is vested in a Working Group
+- Ensure all future Working Groups conform with this process
+
+## Non-goals of the process
+
+- Documenting other governance bodies such as sub-projects or SIGs
+- Changing the status of existing Working Groups/SIGs/Sub-projects
+
+## Is it a Working Group? Yes, if...
+- It does not own any code
+- It has a clear goal measured through a specific deliverable or deliverables
+- It is temporary in nature and will be disbanded after reaching its stated goal(s)
+- It requires input from more than one SIG/sub-project to come up with an adequately-vetted deliverable with reasonable
+  stakeholder consensus
+- It is formulating vendor/implementation-agnostic deliverables
```

Comment:
I'm worried that this is overly broad and can be used to selectively block things. 

As some concrete examples, what if we are working on ARM processor support. That isn't vendor agnostic, and yet it's probably useful and important. Same thing with something like Mellanox high-performance network interconnects, or even NVidia-specific GPU work.

I think we should rephrase this in terms of the behaviors we're hoping to prevent.

Reply:
Deleted